src/rtos/nuttx.c

```diff
@@ -19,52 +19,57 @@
 #include "helper/log.h"
 #include "helper/types.h"
 #include "server/gdb_server.h"
+#include "target/register.h"
+#include "rtos_nuttx_stackings.h"
 
-#include "nuttx_header.h"
-
-
-int rtos_thread_packet(struct connection *connection, const char *packet, int packet_size);
-
-#ifdef CONFIG_DISABLE_SIGNALS
```

Comment:
We may get questions about why we removed these configs. Should we keep them for the other archs?